inheritance/src/test/java/inheritance/ShopTest.java

```diff
@@ -0,0 +1,71 @@
+package inheritance;
+
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class ShopTest {
+
+    @Test
+    public void CansetName() {
```

Comment:
remember to use camelCase -> `canSetName()`

Reply:
ok. Will pay attention on that.